install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -9,6 +9,7 @@ data:
   # storage. This can either be provided with an external kvstore or with the
   # help of cilium-etcd-operator which operates an etcd cluster automatically.
   kvstore: etcd
+  identity-allocation-mode: kvstore
```

Comment:
It's not enough to just add it here, because it's already defined below so now the configmap will have it declared twice. I don't know whether there is any defined precedence for that case. I suggest we instead extend the existing declaration below.

Reply:
Ok I thought about it, let me recheck and see what makes sense to add it in the later part of Configmap.

Reply:
Done.